services/wireguard/endpoint/kernelspace/client.go

```diff
@@ -84,20 +92,46 @@ func (c *client) AddPeer(iface string, peer wg.PeerInfo) error {
 
 func (c *client) up(iface string, ipAddr net.IPNet) error {
 	if d, err := c.wgClient.Device(iface); err != nil || d.Name != iface {
-		if err := exec.Command("ip", "link", "add", "dev", iface, "type", "wireguard").Run(); err != nil {
+		if err := suExec("ip", "link", "add", "dev", iface, "type", "wireguard"); err != nil {
 			return err
 		}
 	}
 
-	if err := exec.Command("ip", "address", "replace", "dev", iface, ipAddr.String()).Run(); err != nil {
+	if err := suExec("ip", "address", "replace", "dev", iface, ipAddr.String()); err != nil {
 		return err
 	}
 
-	return exec.Command("ip", "link", "set", "dev", iface, "up").Run()
+	return suExec("ip", "link", "set", "dev", iface, "up")
+}
+
+func (c *client) addRoute(iface string, endpoint net.UDPAddr) error {
```

Comment:
It's not just addRoute, but it actually adds route to real interface for remote peer, also setups default routing for tunnel - either naming or split this function into two